simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import styles from '../../../styles/homePage.module.scss';
-import { CreatorSlider, JoinerSlider } from '../../common/slider';
```

Comment:
Let's delete the slider folder as well. And remove `react-awesome-slider` from package.json

Reply:
Can u help me when I want to commit I get this.
![image](https://user-images.githubusercontent.com/71724556/94990442-73b11200-057c-11eb-8e16-2b3e79c89150.png)
What should I do?

Reply:
Hi, add these two lines to the top of Burger.jsx to let eslint know you want to suppress these warnings for now.

```
/* eslint-disable jsx-a11y/click-events-have-key-events */
/* eslint-disable jsx-a11y/no-static-element-interactions */

```

We'll get to these later.

Reply:
This should let you commit fine

Reply:
Thanks for the help!